### PR TITLE
(RE-13419) Make SLES a special-case for release packages

### PR DESCRIPTION
**Pull request by e-gris**

### CODE REVIEW

**Comment by e-gris** [comment]
Not yet fully tested but wanted to create the PR in case my style-hack had objections.

File: configs/components/repo_definition.rb [hunk 1]
```diff
@@ -1,31 +1,56 @@
...
-    # way to go if anything else needs to get added here:
-    if platform.is_cisco_wrlinux?
-      repo_path = '/etc/yum/repos.d'
-    elsif platform.is_sles?
-      repo_path = '/etc/zypp/repos.d'
-    else
-      repo_path = '/etc/yum.repos.d'
-    end
+  # Needs to be overriden at the end, sometimes. Deferring now.
+  os_fixup = nil
```
Comment (underscorgan) [inline configs/components/repo_definition.rb:5]: Out of curiousity, why not just set the default here and override it in the case statement where needed?

Comment (e-gris) [reply configs/components/repo_definition.rb]: I swear there was a reason for that. It clearly no longer exists :)

**Comment by e-gris** [comment]
@underscorgan Now I recall why os_fixup needs to be defined at the end. It is dependent on repo_path being defined within the case statement. I moved it back.

**Comment by e-gris** [comment]
Tested on sles-15, fedora-30, & rhel 8

**Pull request merged**